src/lib/components/dropdown/CustomDropdown.svelte

```diff
@@ -0,0 +1,157 @@
+<script lang="ts">
+  import { DownArrow } from "$lib/assets";
+  import { PeopleIcon } from "$lib/assets/app.asset";
+  import { fly } from "svelte/transition";
+
+  export let labelText: string = "Select";
+  export let isRequired: boolean = false;
+  export let inputId: string = "";
+  export let labelDescription: string = "";
+  export let selectInputPlaceholder: string = "Select an option";
+  export let options: any[];
+  export let invalidValue: boolean = false;
+  export let endIconVisible: boolean = false;
+  export let errorText: string = "Invalid Value.";
+  export let handleOnSelect: (value: any) => void;
+
+  let isDropdownVisible: boolean = false;
+  let selectedOption: string = "";
+
+  const closeSparrowDropdown = () => {
+    isDropdownVisible = false;
+  };
+
+  const handleSelectOption = (name: string, e) => {
+    e.stopPropagation();
+    selectedOption = name;
+  };
+
+  const toggleSparrowDropdown = (e) => {
+    e.preventDefault();
+    e.stopPropagation();
+    isDropdownVisible = !isDropdownVisible;
+  };
+</script>
```

Comment:
can we rename it something else rather than customDropdown, as reading the name it is not clear.